src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/Example5.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="LineLength">
+    <property name="ignorePattern" value="^$"/>
```

Comment:
Unfortunately this is mistake in config. Please help us to fix it while I see it.
`<property name="ignorePattern" value="^(import|package) "/>`

Reply:
Done

Reply:
Let me know if we need any content wise change to make it better like adding case for import also or anything else.

Reply:
After reading whole doc  , I noticed phrase 
>By default, package and import statements (lines matching pattern ^(package|import) .*) are not verified by this check.

So this example shows default behavior, that is very not useful for users.

Please add some import that violates length.
Please change this example to  `<property name="ignorePattern" value="^(import) "/>` to avoid violation on imports only. Please update description over example

Reply:
Done review once.